docker-hub/publish/insights-analytics.md

```diff
@@ -10,17 +10,52 @@ to metrics as both raw data and summary data for a desired time span. You can
 view number of image pulls by tag or by digest, and get breakdowns by
 geolocation, cloud provider, client, and more.
 
+## View the analytics data
+
+Analytics data for your repositories is available for viewing on the Insights
+and analytics dashboard at the following URL:
+`https://hub.docker.com/orgs/{namespace}/insights`. The dashboard contains a
+chart visualization of the usage data, as well as a table where you can download
+the data as CSV files.
+
+To view data in the chart:
+
+- Select the data granularity: weekly or monthly
+- Select the time interval: 3, 6, or 12 months
+- Select one or more repositories in the list.
+
+  You can filter the list by repository name.
+
+![Insights and analytics chart visualization](./images/chart.png)
+
+> **Tip**
+>
+> Hovering your cursor over the chart displays a tooltip, showing precise data
+> for points in time.
+
+### Share visualization
+
+You can share the visualization chart with others using the share icon located
+just above the chart:
+
+![Chart share icon](./images/chart-share-icon.png)
+
+Selecting the icon generates a link that gets copied to your clipboard. The link
+preserves the display selections you've made. When someone uses the link, the
+Insights and analytics page opens and displays the chart with the same
```

Comment:
Is this the title of the page in the UI "Insights & analytics" ? If so, we should bold